<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Information</class>
 <widget class="QWidget" name="Information">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>628</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>691</width>
     <height>281</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Process Id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Arrival Time</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Burst Time</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Priority</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Waiting Time</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Turnaround Time</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Response Time</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="gantt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Gantt Chart</string>
   </property>
  </widget>
  <widget class="QLabel" name="gantt_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Average Wait time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="gantt_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Average Turnaroud time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Average response time</string>
   </property>
  </widget>
  <widget class="QLabel" name="avg_reponse_time">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>510</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="avg_turnaroudn_time">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>470</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="average_wait_time">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>440</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
